--- origsrc/gnome-pilot-conduits-2.0.17/email/Makefile.am	2007-01-11 02:31:20.000000000 -0600
+++ src/gnome-pilot-conduits-2.0.17/email/Makefile.am	2010-06-11 18:39:52.418458500 -0500
@@ -8,7 +8,7 @@ email_conduitsdir=$(libdir)/gnome-pilot/
 email_conduits_LTLIBRARIES = \
 	libemail_conduit.la
 
-libemail_conduit_la_LDFLAGS = -module -avoid-version
+libemail_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libemail_conduit_la_SOURCES = 	\
 	email_conduit.c 	\
--- origsrc/gnome-pilot-conduits-2.0.17/expense/Makefile.am	2007-01-11 02:31:21.000000000 -0600
+++ src/gnome-pilot-conduits-2.0.17/expense/Makefile.am	2010-06-11 18:39:56.538694100 -0500
@@ -8,7 +8,7 @@ expense_conduitsdir=$(libdir)/gnome-pilo
 expense_conduits_LTLIBRARIES = \
 	libexpense_conduit.la
 
-libexpense_conduit_la_LDFLAGS = -module -avoid-version
+libexpense_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libexpense_conduit_la_SOURCES = \
 	expense_conduit.c 	\
--- origsrc/gnome-pilot-conduits-2.0.17/mal-conduit/src/Makefile.am	2007-01-11 02:31:21.000000000 -0600
+++ src/gnome-pilot-conduits-2.0.17/mal-conduit/src/Makefile.am	2010-06-11 18:40:20.907087900 -0500
@@ -14,7 +14,7 @@ mal_conduits_LTLIBRARIES = 	\
 	libmal_conduit.la	\
 	$(NULL)
 
-libmal_conduit_la_LDFLAGS = -module -avoid-version
+libmal_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libmal_conduit_la_SOURCES = 	\
 	mal-conduit.h 		\
--- origsrc/gnome-pilot-conduits-2.0.17/memo_file/Makefile.am	2007-01-11 02:31:21.000000000 -0600
+++ src/gnome-pilot-conduits-2.0.17/memo_file/Makefile.am	2010-06-11 18:40:04.994177700 -0500
@@ -11,7 +11,7 @@ memo_file_conduitsdir=$(libdir)/gnome-pi
 memo_file_conduits_LTLIBRARIES = \
 	libmemo_file_conduit.la
 
-libmemo_file_conduit_la_LDFLAGS = -module -avoid-version
+libmemo_file_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libmemo_file_conduit_la_SOURCES = 	\
 	memo_file_conduit.c 		\
--- origsrc/gnome-pilot-conduits-2.0.17/time/Makefile.am	2007-01-11 02:31:21.000000000 -0600
+++ src/gnome-pilot-conduits-2.0.17/time/Makefile.am	2010-06-11 18:40:08.098355300 -0500
@@ -8,7 +8,7 @@ time_conduitsdir=$(libdir)/gnome-pilot/c
 time_conduits_LTLIBRARIES = \
 	libtime_conduit.la
 
-libtime_conduit_la_LDFLAGS = -module -avoid-version
+libtime_conduit_la_LDFLAGS = -module -avoid-version -no-undefined
 
 libtime_conduit_la_SOURCES = 	\
 	time-conduit.c 		\
